x-pack/plugins/cases/public/components/user_action_tree/user_action_host_isolation_comment_event.test.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
+import { HostIsolationCommentEvent } from './user_action_host_isolation_comment_event';
+
+const props = {
+  type: 'isolate',
+  endpoints: [{ endpointId: 'e1', hostname: 'host1' }],
+  href: jest.fn(),
+  onClick: jest.fn(),
+};
```

Comment:
instead of defining a mutable singleton that is shared across tests, can you convert this to a function that returns a new set of props everytime?

```suggestion
const defaultProps = () => {
  return {
    type: 'isolate',
    endpoints: [{ endpointId: 'e1', hostname: 'host1' }],
    href: jest.fn(),
    onClick: jest.fn(),
  }
}
```

then below in your test just do `...defaultProps()` instead of spreading `props`